ingest-app/src/cmr/ingest/data/granule_bulk_update.clj

```diff
@@ -67,10 +69,27 @@
       util/remove-nil-keys
       util/map-keys->kebab-case))
 
+(defn- get-begin-end-dates
```

Comment:
Can you just use the date time parser we already have?  cmr.common.date-time-parser
It provides everything for you and you don't need to do what you are doing in this function.